src/ShortcutGuide.tsx

```diff
@@ -43,7 +43,7 @@ const ShortcutGuide: React.FC = () => {
 						<div className="flex_column">
 							<table role={"group"}>
 								<tbody>
-									<tr className="a11y_empty_row"></tr>
+									<div className="a11y_empty_row"></div>
```

Comment:
Check the html semantics and see if your html code is valid.
Divs inside table elements are a no go.